LocalWiki.php

```diff
@@ -265,6 +265,16 @@ function loadFonts ( OutputPage $out ) {
 	$wgFilterLogTypes['comments'] = false;
 }
 
+if ( $wgDBname === 'samskritavyakaranamwiki' ) {
+	$wgHooks['BeforePageDisplay'][] = 'wfAddSidebarTree';
+	function wfAddSidebarTree( $out, $skin ) {
+	$title = Title::newFromText( 'SidebarTree', NS_MEDIAWIKI );
+	$article = new Article( $title );
```

Comment:
This might not actually work. We don't have access to these classes yet, I dont think.